R/build-articles.R

```diff
@@ -178,7 +178,7 @@ build_article <- function(name,
         theme = NULL
       )
     } else {
-      options <- list()
+      options <- list(keep_tex=FALSE)
```

Comment:
Why shouldn't this be left under the author's control?

Reply:
This is `FALSE` by default, but `TRUE` for `rticles`. So I added this to match pkgdown's default behaviour. I can remove it.